config/initializers/redis.rb

```diff
@@ -11,3 +11,10 @@
     redis: Redis.new(url: IdentityConfig.store.redis_throttle_url),
   )
 end
+
+REDIS_SESSION_POOL_WRAPPER = ConnectionPool::Wrapper.new(
```

Comment:
I was on the fence about defining this here vs in `session_store.rb`

Pros:
- it's a consistent spot we define all our pools

Cons:
- relies on alphabetical ordering of initializers

Reply:
I kinda like it being in session_store.rb, because it keeps all the session_store stuff in one place, but I don't feel strongly about that.

Reply:
That's true! But as a counterpoint, we missed one of our redis connection pools in this other PR because it wasn't defined in the same place: https://github.com/18F/identity-idp/pull/7283#discussion_r1013215884